<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="11" hostname="DESKTOP-E5F8SAC" failures="1" tests="1" name="Default test" time="18.448" errors="0" timestamp="2024-05-23T23:02:35 IST">
  <testcase classname="testCases.LoginTest" name="verifyThetextPasswordFieldisToggledtoHideVisibility" time="2.184">
    <failure type="java.lang.AssertionError" message="expected [********] but found [12451254]">
      <![CDATA[java.lang.AssertionError: expected [********] but found [12451254]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testCases.LoginTest.verifyThetextPasswordFieldisToggledtoHideVisibility(LoginTest.java:196)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyThetextPasswordFieldisToggledtoHideVisibility -->
  <testcase classname="testCases.LoginTest" name="verifyEmailAndPasswordPlaceHolderText" time="0.0">
    <ignored/>
  </testcase> <!-- verifyEmailAndPasswordPlaceHolderText -->
  <testcase classname="testCases.LoginTest" name="loginWithInactiveCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithInactiveCredentials -->
  <testcase classname="testCases.LoginTest" name="loginWithoutCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithoutCredentials -->
  <testcase classname="testCases.LoginTest" name="loginWithInvalidUserAndValidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithInvalidUserAndValidPassword -->
  <testcase classname="testCases.LoginTest" name="loginWithValidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithValidCredentials -->
  <testcase classname="testCases.LoginTest" name="verifyLogginInToApplicationAndBrowserBackOptn" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLogginInToApplicationAndBrowserBackOptn -->
  <testcase classname="testCases.LoginTest" name="loginWithInvalidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithInvalidCredentials -->
  <testcase classname="testCases.LoginTest" name="verifyNumberofUnsuccessfulLoginAtepmts" time="0.0">
    <ignored/>
  </testcase> <!-- verifyNumberofUnsuccessfulLoginAtepmts -->
  <testcase classname="testCases.LoginTest" name="verifyForgotPasswordOptionLink" time="0.0">
    <ignored/>
  </testcase> <!-- verifyForgotPasswordOptionLink -->
  <testcase classname="testCases.LoginTest" name="loginWithValidUserAndinvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- loginWithValidUserAndinvalidPassword -->
  <testcase classname="testCases.LoginTest" name="verifyLogoutFromApplicationAndBrowserBackOptn" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLogoutFromApplicationAndBrowserBackOptn -->
</testsuite> <!-- Default test -->
